source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -51,6 +51,15 @@ import { CardObjectRegistry, GlobalRegistry, ElementSingletonBehavior } from "./
 import { Strings } from "./strings";
 import { MenuItem, PopupMenu } from "./controls";
 
+const ttPolicy = window.trustedTypes?.createPolicy('adaptivecards', {
+    createHTML: value => value,
+});
+
+function clearElement(element: HTMLElement) : void {
+    const trustedHtml = ttPolicy?.createHTML("") ?? "";
+    element.innerHTML = trustedHtml as string;
+}
```

Comment:
realizing a bit late that this should probably be:

```suggestion
function clearElement(element: HTMLElement) : void {
    const trustedHtml = window.trustedTypes?.emptyHTML ?? "";
    element.innerHTML = trustedHtml as string;
}
```